pkg/datapath/linux/datapath.go

```diff
@@ -43,6 +44,12 @@ func NewDatapath(config DatapathConfiguration) datapath.Datapath {
 
 	dp.node = NewNodeHandler(config, dp.nodeAddressing)
 
+	if config.EncryptInterface != "" {
+		if err := connector.DisableRpFilter(config.EncryptInterface); err != nil {
+			log.Warn("Rpfilter could not be disabled on %s, node to node encryption may fail", config.EncryptInterface)
```

Comment:
```suggestion
			log.WithField(logfields.Interface, config.EncryptInterface).Warn("Rpfilter could not be disabled, node to node encryption may fail")
```

Reply:
Went ahead and folded this in thanks for the hint.